docs/integrate-cardano/overview.md

```diff
@@ -13,9 +13,24 @@ image: /img/og/og-developer-portal.png
 ## Introduction
 Here we show you how to integrate Cardano into existing websites and services. 
 
-## Topics
+## Integration Components
+- [Overview](/docs/get-started/cardano-components.md) of the different Cardano commponents.
+- [cardano-node](https://github.com/input-output-hk/cardano-node) is the top level for the node and aggregates the other components from other packages: consensus, ledger and networking, with configuration, CLI, logging and monitoring.
+- [cardano-wallet](https://github.com/cardano-foundation/cardano-wallet) helps you manage ada. You can use it to send and receive payments on the Cardano blockchain via a http and cli interface.
```

Comment:
Not sure if cardano-wallet is a core component anymore

Reply:
Is this resolved?

Reply:
I think this tiebreaker should be that we have a component page still dedicated to it here on the Portal 🤓 _Get Started > Cardano Components > Installing cardano-wallet_